docs/how-tos/working-with-postgresql-data-types.rst

```diff
@@ -461,6 +465,80 @@ Finally, do the request :ref:`casting the range column <casting_columns>`:
 
       create cast (mytsrange as json) with function mytsrange_to_json(mytsrange) as assignment;
 
+Binary Data Types
+-----------------
+
+To send raw binary to PostgREST you need a function with a single unnamed parameter of `bytea type <https://www.postgresql.org/docs/current/datatype-binary.html>`_. For example, let's create a table that will save some files and a function that inserts data to that table:
+
+.. code-block:: postgres
+
+   create table files (
+     id serial primary key,
+     file bytea
+   );
+
+   create function upload_binary(bytea) returns void as $$
+     insert into files (file) values ($1);
+   $$ language sql;
+
+.. TODO: GRANT FOR SERIAL
```

Comment:
You can use the `generated always as identity` instead of `serial`.

Reply:
Didn't know that it's even suggested not to use `serial`: https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial